Modules/Core/Tests/Feature/ImportControllerTest.php

```
@@ -35,4 +37,368 @@ public function it_displays_import_page(): void
+            'btn_submit' => '1',
+            'files' => ['clients.csv', 'malicious.php', 'invoices.csv'],
+        ];
+
+        /** Act */
+        $this->actingAs($user);
+        $response = $this->post(route('import.form'), $formData);
+
+        /** Assert */
+        $response->assertRedirect(route('import.index'));
+        // Should only process clients.csv and invoices.csv, ignore malicious.php
+    }
```

PHPMD (2.15.0)

The method it_filters_out_disallowed_files_on_import is not named in camelCase. (undefined)

(CamelCaseMethodName)